includes/templates/responsive_classic/templates/tpl_modules_products_new_listing.php

```diff
@@ -36,19 +51,19 @@
         $display_products_name = '';
       }
 
-      if (PRODUCT_NEW_LIST_MODEL != '0' and zen_get_show_product_switch($products_new->fields['products_id'], 'model')) {
+      if (PRODUCT_NEW_LIST_MODEL != '0' and $flag_show_model) {
         $display_products_model = '<b>' . TEXT_PRODUCT_MODEL . '</b>' . $products_new->fields['products_model'] . str_repeat('<br>', substr(PRODUCT_NEW_LIST_MODEL, 3, 1));
       } else {
         $display_products_model = '';
       }
 
-      if (PRODUCT_NEW_LIST_WEIGHT != '0' and zen_get_show_product_switch($products_new->fields['products_id'], 'weight')) {
+      if (PRODUCT_NEW_LIST_WEIGHT != '0' and $flag_show_weight) {
```

Comment:
Should all these 'and' connectors be changed to `&&`?  I thought that we were trying to move away from `and` and `or` in preference to `&&` and `||`.

Reply:
Just trying to keep the scope of this change limited.  What you are suggesting is a good idea, I just didn't do it in this change.